readthedocs/projects/querysets.py

```diff
@@ -131,27 +131,31 @@ def prefetch_latest_build(self):
         """
         from readthedocs.builds.models import Build
 
-        # Prefetch the latest build for each project.
-        subquery_build_latest = Subquery(
-            Build.internal.filter(project=OuterRef("project_id"))
-            .order_by("-date")
-            .values_list("id", flat=True)[:1]
+        # Get most recent and recent successful builds
+        builds_latest = (
+            Build.internal.filter(project__in=self)
```

Comment:
Instead of `__in` can't we just use `project__pk=self.pk` here as I did in another PR? That worked pretty good there.

Reply:
`self` is a `Project.objects` queryset, not an individual model instance.